### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by nikeshashar**
Updating FAQ without a ticket

### CODE REVIEW

File: data/faqs.yml [hunk 1]
```diff
@@ -155,13 +155,13 @@
...
 
     - question: What if I don't have the deposit?
       answer: "Once a student is accepted on to the course, their spot will be guaranteed until 28 days before the course start-date. In this time the seat will not be offered to anyone else. If the deposit is not paid within 28 days of the course start-date, the spot will be opened up to students on the waiting list. While this doesn't necessarily mean that you will lose your seat, so long as you have not paid your deposit you're liable to be moved to a later cohort at any time."
 
     - question: Do I need to buy books?
-      answer: "Everything you need will be provided by Makers Academy once you enroll."
+      answer: "Books are kind of outdated in the fast pacing world of software development. You won't need any on the course but we can recommend some if you're really after some physical materials."
```
Comment (spike01) [inline data/faqs.yml]: fast-paced

Comment (spike01) [inline data/faqs.yml]: :joy: @ "physical materials"

File: data/faqs.yml [hunk 2]
```diff
@@ -64,21 +64,21 @@
     - question: If I already know how to program a little, is Makers Academy for me?
       answer: "Absolutely. Makers Academy is more than just learning how to program — it's learning how to be a developer. It's learning how to build your own prototypes. It's learning where to go to find answers. It's learning about what the atmosphere of working in a startup is really like. It's learning to be a Maker from some of the best Makers in the world."
 
+    - question: Do you consider applications from outside the EU?
+      answer: "Absolutely. We’ve had students from all over Europe, the USA, Singapore, Hong Kong, South Africa, Pakistan, Australia, New Zealand and even Venezuela!"
```
Comment (spike01) [inline data/faqs.yml]: Do we need all the exclamation marks!

**Comment by spike01** [comment]
Ship it if you read my notes!

File: data/faqs.yml (hunk 2 above)
Comment (nikeshashar) [inline data/faqs.yml]: Removed

**Comment by nikeshashar** [comment]
Should be sorted now @spike01

**Pull request merged**